src/main.js

```diff
@@ -62,23 +65,31 @@ function formValidation(formInput) {
     confirmValid = true;
   }
   return confirmValid;
-}
+};
 
 function clearForm() {
   formTitle.value = "";
   formBody.value = "";
-}
+};
 
 function show(element) {
   element.classList.remove('visibility-hidden');
-}
+};
```

Comment:
I didn't realize you like to add a semicolon after every function - noted for the next stuff!